packages/@react-aria/tooltip/docs/useTooltipTrigger.mdx

```diff
@@ -112,7 +112,7 @@ function Example(props) {
 
   return (
     <span style={{position: 'relative'}}>
-      <button ref={ref} {...triggerProps}>I have a tooltip</button>
+      <button ref={ref} state={state} {...triggerProps}>I have a tooltip</button>
```

Comment:
I think you meant to add this to `Tooltip`?

Reply:
yeah, this was done in a hurry 🤦 fixed